Podcast/ShareEpisodeViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ShareEpisodeViewController.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/15/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+import UIKit
+
+class ShareEpisodeViewController: FollowerFollowingViewController  {
+
+    var episode: Episode
+
+    init(user: User, episode: Episode) {
+        self.episode = episode
+        super.init(user: user)
+        followersOrFollowings = .Followers
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "Tap to Share"
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let tappedUser = users[indexPath.row]
+        let shareEpisodeOption = ActionSheetOption(type: .confirmShare, action: {
+            self.episode.share(with: tappedUser, success: { self.navigationController?.popViewController(animated: true)}, failure: nil) // TODO: error handling
```

Comment:
it would be nice to have a brief notification/something pop up to show that it was a success/failure but i know that's a todo

Reply:
I can add this!